### PR TITLE
fix: remove padded zws fields on github command

### PR DESCRIPTION
**Pull request by onerandomusername**
#### Before:
image
#### After:
image

### CODE REVIEW

**Review by Arnav-2004 — APPROVED** [review]
LGTM!

**Comment by gustavwilliam** [comment]
Have you viewed it on desktop?

File: bot/exts/github/_profile.py [hunk 1]
```diff
@@ -60,17 +59,13 @@ def get_data(username: Optional[str], user_data: dict, org_data: dict) -> Embed:
...
                 value=f"[{user_data['public_gists']}](https://gist.github.com/{username})",
             )
-            embed.add_field(
-                name="Organizations",
-                value=" | ".join(orgs) if orgs else "No Organizations",
-            )
-            embed.add_field(name="\u200b", value="\u200b")
             embed.add_field(name="Website", value=blog)
+            if orgs:
+                embed.add_field(name="Organizations", value=" | ".join(orgs))
```
Comment (Arnav-2004) [inline bot/exts/github/_profile.py:68]: The `Organizations` field will only show if someone is in an organization. If that's fine with you @gustavwilliam

Comment (gustavwilliam) [reply bot/exts/github/_profile.py:68]: Yeah sure

**Comment by gustavwilliam** [comment]
I don't understand what the fields were put there for. But let's try this.

**Pull request merged**